packages/kbn-search-api-panels/components/select_client.tsx

```diff
@@ -97,26 +97,23 @@ export const SelectClientPanel: FC<PropsWithChildren<SelectClientPanelProps>> =
   return (
     <OverviewPanel
       description={
-        <FormattedMessage
-          id="searchApiPanels.welcomeBanner.selectClient.description"
-          defaultMessage="Elastic builds and maintains clients in several popular languages and our community has contributed many more. Select your favorite language client or dive into the {console} to get started."
-          values={{
-            console: (
-              <TryInConsoleButton
-                application={application}
-                consolePlugin={consolePlugin}
-                sharePlugin={sharePlugin}
-                content={i18n.translate(
-                  'searchApiPanels.welcomeBanner.selectClient.description.console.link',
-                  {
-                    defaultMessage: 'Console',
-                  }
-                )}
-                link
-              />
-            ),
-          }}
-        />
+        <EuiFlexGroup direction="column" alignItems="flexStart" justifyContent="flexStart">
+          <EuiFlexItem>
+            <FormattedMessage
+              id="searchApiPanels.welcomeBanner.selectClient.description"
+              defaultMessage="Elastic builds and maintains clients in several popular languages and our community has contributed many more. Select your favorite language client or dive into the console to get started."
+            />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <TryInConsoleButton
+              application={application}
+              consolePlugin={consolePlugin}
+              sharePlugin={sharePlugin}
+              type="button"
```

Comment:
Wouldn't it make more sense to keep this at least visually link (which is how we designed it) and add more context with an aria-label?

Reply:
@dave-gus ^^^ what do you think?

Reply:
@1Copenut, could you please take a look as well? I replaced it with a button because I believe this option is more preferable from a ScreenReader perspective. This way, the text is not interrupted by reading interactive element. But, I am open to any version you suggest.

Reply:
I agree with @alexwizp that this is better served as a button. With respect to accessibility, we recommend links "go" somewhere and buttons be used to activate overlays, menus, and in this case the console that lives at the bottom of the viewport.

Reply:
@1Copenut  I think the main question here is whether the button element should be inside the text block or not. Adding more ARIA information will make the main text less understandable because the context will be switched to the button.

I recommend to do like in that PR. Keep text separate of button.

Reply:
@alexwizp I recommend the button be outside of the text block, as you have it here. Can we modify the `aria-label` slightly? Proposing the following:

```diff
! The suggested change is the visible label plus the explainer about embedded console.
! Roughly the same message with fewer words

- aria-label="Try the snipped in the Console - opens in embedded console"
+ aria-label="Try in Console - opens in embedded console"
```

Reply:
done!